RecoHI/HiTracking/python/HITrackingRegionProducer_cfi.py

```diff
@@ -88,3 +88,15 @@
 HiTrackingRegionFactoryFromSTAMuonsBlock.MuonTrackingRegionBuilder.vertexCollection = cms.InputTag("hiSelectedVertex")
```

Comment:
@echapon Let me extend the confirmation question also to `RecoHI/HiMuonAlgos/python/HiRegitMuonTobTecStep_cff.py` which I see was disabled in the same commit (https://github.com/cms-sw/cmssw/commit/a905b38b6c33b9026c1cbaab9cb3d4a42cd0f415) and appears to be unused.